app/utils/data_access.py

```diff
@@ -769,21 +769,21 @@ def form_to_individual(form, user=None):
     # check if the individual exists in the datbase
     with DATABASE.atomic():
         try:
-            individual = Individual.get(Individual.number == form["number"])
+            individual = Individual.get(Individual.id == form["id"])
         except DoesNotExist:
             individual = Individual()
 
-    # If the form has an id - make sure that it points to the same individual as
-    # the number.
-    if "id" in form and form["id"] != individual.id:
-        raise ValueError("Number can not be updated")
-
     can_manage = user and (
         user.is_admin
         or user.is_manager
         and individual.current_herd.genebank_id in user.is_manager
     )
 
+    # If the form has an number - make sure that it points to the same individual as
+    # the number if not manager then you are allowed to update number.
+    if not can_manage and "number" in form and form["number"] != individual.number:
```

Comment:
This will not work when adding new animals. So its need to bee rewritten or change logic